Azure/Dev/resource_aks.tf

```diff
@@ -28,23 +28,21 @@ resource "azurerm_container_registry" "acr" {
 resource "azurerm_kubernetes_cluster" "aks" {
   name                            = "${var.project}-${var.environment}-aks"
   resource_group_name             = azurerm_resource_group.aks.name
-  kubernetes_version              = var.aksversion
+  kubernetes_version              = "1.16.10" #var.aksversion
   location                        = var.region
   dns_prefix                      = "${var.project}${var.environment}aksdns"
   node_resource_group             = "${var.project}-${var.environment}-rg-aks-pool"
 
   default_node_pool {
-    name                          = "devpool1"
-    vm_size                       = var.vm_size #"Standard_B4ms"
-    #os_disk_size_gb               = 30
-    vnet_subnet_id                = azurerm_subnet.aks.id
+    name                          = "devpool"
+    vm_size                       = var.vm_size
+    vnet_subnet_id                = azurerm_subnet.aks_nodes.id
     type                          = "VirtualMachineScaleSets"
-    enable_auto_scaling           = "false"
-    max_pods                       = 30
-    #max_count                     = 6
-    #min_count                     = 4 
-    node_count                    = 4
-   #enable_node_public_ip         = "true"
+    enable_auto_scaling           = "true"
+    max_pods                      = 110
+    max_count                     = 4
+    min_count                     = 2 
+    node_count                    = 4 
```

Comment:
should node count be excluded, as it'll vary wit the autoscaling?

Reply:
Excluded now - good spot